pkg/osgi_config.go

```diff
@@ -13,25 +14,38 @@ import (
 type OSGiConfig struct {
 	manager *OSGiConfigManager
 	pid     string
+	fpid    string
+
+	// constant ID used to find config later (as AEM is generating random PID when adding new config which cannot be
+	// used to achieve idempotency)
+	cid string
 }
 
 func (c OSGiConfig) Pid() string {
```

Comment:
in Go... abbrevations should be uppercase; 

`ID()` `FPID()` `CID()` `makeThisHTTPCallPlease()` etc ;)

Reply:
It's been this way when I came :P